src/components/Sublist3r/Sublist3r.tsx

```diff
@@ -0,0 +1,193 @@
+import { useState, useEffect, useCallback } from "react";
+import { Button, Stack, TextInput, Checkbox } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { RenderComponent } from "../UserGuide/UserGuide";
+import InstallationModal from "../InstallationModal/InstallationModal";
+import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
+
+/**
+ * Represents the form values for the Sublist3r component.
+ */
+interface FormValuesType {
+    targetDomain: string;
+}
+
+/**
+ * The Sublist3r component.
+ * @returns The Sublist3r component.
+ */
+const Sublist3r = () => {
+    // Component state variables
+    const [loading, setLoading] = useState(false); // State variable to indicate loading state
+    const [output, setOutput] = useState(""); // State variable to store the output of the command execution
+    const [allowSave, setAllowSave] = useState(false); // State variable to allow saving of output
+    const [hasSaved, setHasSaved] = useState(false); // State variable to indicate if output has been saved
+    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
+    const [opened, setOpened] = useState(!isCommandAvailable); // State variable that indicates if the modal is opened.
+    const [loadingModal, setLoadingModal] = useState(false); // State variable to indicate loading state of the modal.
```

Comment:
I think this probably needs to be set as true. I am finding that your commandchecker isn't quite running as expected. It seems to be briefly popping up every time I open the tool. Please test and correct if required. 
```suggestion
    const [loadingModal, setLoadingModal] = useState(true); // State variable to indicate loading state of the modal.
```